packs/trident-operator-v23.01.0/pack.json

```diff
@@ -0,0 +1,17 @@
+{
+  "addonType":"system app",
+  "cloudTypes": [
+    "vsphere"
```

Comment:
it's not limited to vsphere, but has only been tested on that, but it should work on pretty much all providers. I can add all clouds for now, because you need netapp hardware for this and won't be using this if you don't know how to use it.